internal/rootcoord/create_collection_task.go

```diff
@@ -439,17 +439,61 @@ func (t *createCollectionTask) appendSysFields(schema *schemapb.CollectionSchema
 }
 
 func (t *createCollectionTask) prepareSchema(ctx context.Context) error {
```

Comment:
we need some case to guarantee that even we add new field to schema the collection is only different for system field ids?